### ISSUE TITLE
MiniHud causes massive lag spikes

### ISSUE THREAD

**Issue by MateoF24** [sakura-ryoko/minihud#71]
![2024-11-20_14 32 03](https://github.com/user-attachments/assets/b7e19e9b-743a-4fe7-84d5-01297dae4dca)

- [x] Mod Version: minihud 0.33.0-sakura.8
- [x] Malilib Version: malilib 0.22.0-sakura.7
- [x] Fabric-api Version: 0.109.0+1.21.3
- [x] Fabric Loader Version: 0.16.9
- [x] Minecraft Version: 1.21.3

**Steps to try to reproduce:**

1. Create a new Default World
2. Fly around to generate new chunks
3. Lag spikes appear (only when generating new chunks!)

**Description:**

In vanilla or with only malilib and fabric api installed these lag spikes do not occur, mods like litematica, servux, syncmatica and the rest of the Masa's mods do not cause lag, optimization mods such as Sodium, Lithium or experimental ones such as C2ME do not appear to solve or benefit the bug.

I have read the entire log even with all the debugging functionalities that malilib has but there are no indications of errors or warnings. Spark mod also did not allow me to analyze what was causing the failure, it took me a long time to identify that MiniHud was the cause, since I individually tested each mod that I had installed until I found it. I have no idea why it happens, since in previous versions (<=1.20.4) I had no errors.

Sorry for my bad English, it is not my native language.

**Comment by sakura-ryoko** [comment]
Unable to reproduce.  Perhaps you have something else going on, such as using the Debug Rendering Service being enabled; which generates lots of extra Vanilla debug packets.  In my own testing I haven't noticed much difference.

**Comment by MateoF24** [comment]
I solved it by resetting the MiniHud configurations to default, perhaps the problem was due to the fact that when updating the version I kept the configuration I had previously (MiniHud Config for mc 1.20.4).
Or at least I think that was it, since it was the only thing I had not restored when updating the version because I wanted to keep my configurations and now I was trying a completely clean installation and it worked fine! 

Sorry for the inconvenience, have a nice day!
